getting-started/gopher-run/cmd/main.go

```diff
@@ -54,6 +64,46 @@ func main() {
 	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
 }
 
+// Concatenate data from the top runs and start a Cloud ML training job on them
+func submitTrainingJob(a *app) {
```

Comment:
Make this a method on `*app` and it should probably return an error.

```suggestion
func (a *app) submitTrainingJob() error {
```